src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -20,35 +20,36 @@
 public class LogsCenter {
```

Comment:
The commit message is still a bit imprecise for my liking. Is the following a good fit?
```
The current way of initializing loggers causes all loggers to
default to INFO level, even if a different level is specified
via the config.json or programmatically.

Let's create a logger named `ab3` to serve as the base logger, and 
automatically prefix all other loggers with `ab3.` so that they all
become descendants of the base logger. This allows us to change
the logging level of all loggers simply by changing the logging
level of the base logger (unless those loggers have specified
their own logging level).

Implications: Loggers not created using the LogCenter#getLogger(...)
methods will not come under the above control mechanism, unless
their name start with `ab3.`.
 ```